docs/index.rst

```diff
@@ -315,3 +315,43 @@ Configuration Options
   * Type: long
   * Default: 60000
   * Importance: low
+
+
+Schema Evolution
+----------------
+
+The JDBC connector supports schema evolution. When there is a change in a database table schema,
+the JDBC connector can detect the change, create a new Kafka connect schema and try to register
+a new Avro schema in the schema registry. Whether we can successfully register the schema or not
+depends on the compatibility level of the schema registry which is backward by default.
+
+For example, if we remove a column from the table, the change is a backward compatible and the
+corresponding Avro schema can be successfully registered in the schema registry. If we modify
+the database table schema to change a column type or add a column, when the Avro schema is
+registered to the schema registry, it will be rejected as the changes are not backward compatible.
+
+You can change the compatibility level of schema registry to allow non compatible schema or other
+compatibility levels. There are two ways to do this:
+
+* Configure the schema registry to use other schema compatibility level by setting
+  ``avro.compatibility.level`` in schema registry. Note that this is a global setting that applies
+  to all schemas in the schema registry.
+
+* Set the compatibility level for subjects which are used by the connector using
```

Comment:
We should probably put this one first since it is really the preferable way to change this setting.